### DISCUSSION TITLE
Install from git

### DISCUSSION THREAD

**Discussion by cspotcode** [TypeStrong/ts-node#1977] [Official Recipes]
To use pre-release features if ts-node, you can install directly from git:

```
npm install -D TypeStrong/ts-node#main
```

This method allows installing any branch, tag, or pull request.

**Comment by ghost** [comment]
This might not currently work properly on Yarn Berry 🤔 #2002

**Comment by yeliex** [comment]
@cspotcode when would the latest version release, the build environment in our production cannot visit github

**Comment by RobinTail** [comment]
@cspotcode , could you please release a beta version instead, using `npm publish --tag beta` or similar?
